independent-publisher-connectors/AzureADApplications/apiDefinition.swagger.json

```diff
@@ -49,23 +54,60 @@
             "x-ms-summary": "Search criteria",
             "x-ms-visibility": "advanced"
           },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Filters results (rows)",
+            "x-ms-summary": "Filter criteria",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "ConsistencyLevel",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "Set this property to 'eventual' if you are using $search",
+            "x-ms-summary": "Enable advanced queries",
+            "x-ms-visibility": "advanced",
+			"enum": [
+              "eventual",
+              ""
+            ]
+
+          },		  
           {
             "name": "$count",
             "in": "query",
             "required": false,
             "type": "string",
             "description": "true or false - Retrieves the total count of matching resources",
             "x-ms-summary": "Display count",
-            "x-ms-visibility": "advanced"
+            "x-ms-visibility": "advanced",
+			"default" : "true",
+			"enum": [

```

Comment:
@paulculmsee The list of enum values has been added to the input field. Existing flows may use values outside of this list. can you please confirm that no users will be affected by this change?

Reply:
I don't believe there will be an impact as users were not having success with the connector and there are only two possible values. However, I just noticed that I left in a policy template that was setting this header so I need to remove it and retest. So I will push an update out tomorrow morning

Reply:
Left policy template in and retested. Relates to ms graph API advanced search queries. This enum has been removed altogether